src/openforms/submissions/models/submission_report.py

```diff
@@ -74,7 +75,7 @@ def generate_submission_report_pdf(self) -> str:
             )
             self.content = ContentFile(
                 content=pdf_report,
-                name=f"{form.name}.pdf",  # Takes care of replacing spaces with underscores
+                name=f"{slugify(form.name)}.pdf",
```

Comment:
is there a reason you use `slugify` rather than using `form.slug` for this?

Reply:
I thought it would be better in case the form name has been updated. Should this be consistent only with the `form.slug`?

Reply:
I think it makes little difference - the form slug is visible in the browser URL anyway. The filename of the PDF is just some courtesy, either approach works for me!